### PR TITLE
HTML Validation and other stuff

### PR DESCRIPTION
**Pull request by aayn**

### CODE REVIEW

File: .idea/workspace.xml [hunk 1]
```diff
@@ -0,0 +1,281 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (ParthKolekar) [inline .idea/workspace.xml]: @aayn 

Please remove your IDE specific files.

File: 13/main/index.html [hunk 2]
```diff
@@ -0,0 +1,1480 @@
...
+
+  }
+  function postQuestion()
+  {
+
+  question = document.getElementById("question_box").value;
+FB.api(
+  '/me/felicity-chatbot:ask',
+  'post',
+  {
```
Comment (ParthKolekar) [inline 13/main/index.html]: We can ignore this part for now as the chatbot is not yet ready.

File: 13/main/index.html [hunk 3]
```diff
@@ -0,0 +1,1480 @@
...
+  }
+  </script>
+
+
+
+<title> Felicity 2k13  </title>
+<link rel="stylesheet" type="text/css" href="style/style.css" media="all" />
+<link rel="stylesheet" type="text/css" href="style/fb_style.css" media="all" />
+<script type="text/javascript" src="js/modernizr.js"></script>
+<script type="text/javascript" src="js/get_answer_GHJGDJHGJDHG.js"></script>
```
Comment (ParthKolekar) [inline 13/main/index.html]: I think the get_answer_<string> is a randomly generated string.

But I suppose that this would do for now.

File: 13/main/index.html [hunk 4]
```diff
@@ -0,0 +1,1480 @@
...
+<script>
+//alert("This is a secretive development. DONOT share the link until a formal website release");
```
Comment (ParthKolekar) [inline 13/main/index.html]: True That.

File: 13/main/index.html [hunk 5]
```diff
@@ -0,0 +1,1480 @@
...
+        <div class="unit static threeWide" id="Logo" style="height: 120px">
+
+
+
+
+ <div>
+    <marquee  id="marqueedisp" scrollamount="3" onmouseout="this.scrollAmount=3" onmouseover="this.scrollAmount=0" direction="left" class="list_marquee" >
+    Felicity Regrets Hyderabad Bomb Blasts, Fest continues.
+    &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;
+    Paper presentation event has started. <a href="http://feli.co/paper-ppt">feli.co/paper-ppt </a>
```
Comment (ParthKolekar) [inline 13/main/index.html]: We are no longer at feli.co

feli.co was the felicity domain for one year. 

@polybuildr 
@nisargjhaveri 
@hharchani 

Please define rule for feli.co -> felicity.iiit.ac.in

File: 13/main/index.html [hunk 6]
```diff
@@ -0,0 +1,1480 @@
...
+                        </div>
+                </div>
+        </div>
+
+
+
+
+
+
+<!-- Pulsation -->
```
Comment (ParthKolekar) [inline 13/main/index.html]: @nisargjhaveri 
@polybuildr 
@hharchani 

I now know where you got your "One Page To Rule Them All" idea. :grin:

File: 13/main/index.html [hunk 7]
```diff
@@ -0,0 +1,1480 @@
...
+                        <div class="column">
+                <span id="ZombieZone_image2"><img alt="literary" src="images/literary.jpg" /></span>
+                        </div>
+                        <div class="column-two last" >
+                                <h3 id="ZombieZone_header1">Literary Cafe</h3>
+                                <p id="ZombieZone_tagline1" class="tagline">Words are words</p>
+                                <p class="bottomSection" id="kalakshetra_content2">
+<h3>Parley(GD) Saturday</h3>
+<p>
+What did you feel when you saw Afzal's softer side? Do you really think it took a real shock for Indian feminists to rise? Did you feel like making your vote count when you saw Modi or Rahul speak? During your college days, did you ever feel like raising your voice on the polarized Indian education system? Here's your chance, as Felicity presents 'Parley', to speak up your thoughts and opinions.
```
Comment (ParthKolekar) [inline 13/main/index.html]: WTF?

File: 13/main/index.html [hunk 8]
```diff
@@ -0,0 +1,1480 @@
...
+                <span id="ZombieZone_image3"><img alt="advt" src="images/advt.jpg" /></span>
+
+
+                                <br />
+                                <br />
+                                <br />
+Felicity will be hosting Skateboarding Workshop and rappelling. We aim at getting you all geared up for those height with rapelling and slam some of those floors with your skateboard skills. Organized in association with GHAC. Registration onsite, 22, 23, 24 February.
+                                <br />
+                                <br />
+                                <br />
```
Comment (ParthKolekar) [inline 13/main/index.html]: :grin: 

LOL @ the <br/>

File: 13/main/index.html [hunk 9]
```diff
@@ -0,0 +1,1480 @@
...
+			<p class="tagline">We'll be glad to hear from you.</p>
+			</div>
+			<div class="column-two">
+                                <ul>
+                                    <li> Prasanth Garapati : +91 93 97 97 97 49 </li>
+                                    <li> Nishith Maheshwari : +91 77 99 08 77 31 </li>
+                                    <li> SP Mohanty : +91 81 42 09 26 36 </li>
+                                </ul>
+                                <p>
+                                     Official Channel : felicity [at] students [dot] iiit [dot] ac [dot] in
```
Comment (ParthKolekar) [inline 13/main/index.html]: @polybuildr 
@nisargjhaveri 
@hharchani 

Add real contacts? 

Add warnings that this is no longer real page?

File: 13/main/index.html [hunk 10]
```diff
@@ -0,0 +1,1480 @@
...
+
+                                <span class="event_name"> Matchitecture</span>
+                                <span class="bottomSection hidden" id="match_content" style="display:none">
+This fun and creative event involves building your dream house- with a twist. Participants can only use matchboxes.. and no adhesive!
+In one hour, you can relive your childhood of building sandcastles using matchsticks and boxes instead.
+Participants are provided with 50-60 matchboxes and nothing else to create an eclectic structure of their own imagination.
+Creations are judged based on the length, creativity and stability of the structure. Out of the box thinking is always appreciated.
+                                <br />   </span>
+                                <span class="hidden" id="match_header">  Matchitecture </span>
+                                <span class="hidden" id=
```
Comment (ParthKolekar) [inline 13/main/index.html]: @nisargjhaveri 
@polybuildr 
@hharchani 

We need to change this?

**Comment by ParthKolekar** [comment]
Overall,

Damn Good Job. :+1: . Nice.

@polybuildr 
@nisargjhaveri 
@hharchani 

Please look at my comments and respond.

LGTM

**Comment by polybuildr** [comment]
Wow, @ParthKolekar saying you did a good job means you did a really good job. I'll review this first chance I get in the morning. Sorry for the delay!

Thanks for the PR. :D Should {resolve T12}.

File: .idea/workspace.xml (hunk 1 above)
Comment (polybuildr) [inline .idea/workspace.xml]: Yeah, I don't know where this came from, but it certainly shouldn't be there.

Comment (ParthKolekar) [inline .idea/workspace.xml]: This is IntelliJ.

File: 13/main/index.html (hunk 4 above)
Comment (polybuildr) [inline 13/main/index.html]: xD

File: 13/main/index.html (hunk 5 above)
Comment (polybuildr) [inline 13/main/index.html]: I think 404s are okay. Their URL structure was pretty weird. I'd prefer letting go of the past instead of trying to maintain an archive in this case.

File: 13/main/index.html (hunk 9 above)
Comment (polybuildr) [inline 13/main/index.html]: We'll add a conspicuous warning to all old static sites.

File: 13/main/index.html (hunk 10 above)
Comment (polybuildr) [inline 13/main/index.html]: I didn't exactly get what you're talking about here.

File: 13/main/index.html [hunk 11]
```diff
@@ -0,0 +1,1480 @@
...
+		<h4 class="infoHeader"  >Detailed Schedule</h4>
+
+		<div class="content cf">
+			<div class="column-three last">
+			</div>
+			<div class="column-two">
+                        <br /><br/><br />
+				<p>
+<br /><br /><br /><br />
+<a href="http://10.4.3.65/13/Misc/images/pass_new_back.jpg" target="_blank"> Click here for a detailed schedule </a>
```
Comment (polybuildr) [inline 13/main/index.html]: We really don't want any IPs turning up in this version. I didn't find the image in a quick search, so I'm okay with just editing the URL to make it relative and letting it 404.

**Comment by polybuildr** [comment]
Sorry for the late review.

A few higher-level things:
- Please change the directory structure such that dropping this repository into `felicity.iiit.ac.in/13/` sets everything up for us. As in, there shouldn't be a `/13` in this repo.
- I agree with @ParthKolekar. Let's get rid of the chatbot for now, at least. Which means we can get rid of the `Web2py` directory, and remove any obvious and easy-to-remove references to the chatbot code.

@ParthKolekar, @nisargjhaveri, @hharchani: I think we need to move on from this now. As long as the site is reasonably functional, let's just merge and move on.

@aayn, great work. :D Please fix the issues we commented on inline (as long as it can be completed in a reasonable amount of time - let us know if not) and please fix the directory structure and get rid of the chatbot.

After that, we'd prefer this PR to be updated with squashed commits, and then we can merge. :D

File: 13/main/index.html (hunk 10 above)
Comment (ParthKolekar) [inline 13/main/index.html]: Update GA account? 

:grin:

Comment (polybuildr) [inline 13/main/index.html]: Okay, this is really weird. In my emails and on the main PR page, the relevant lines shown are completely unrelated - which is why I didn't get what you were talking about.

Also, how about we leave this? :P I don't see much point of tracking these, and let the old guys see if they ever open GA? :P

Comment (nisargjhaveri) [inline 13/main/index.html]: Let's track these also, please? We can change these or maybe we can add that code in the notice we are planning to put. (I don't know what happens if there are more than one GA on a page, someone please check?)

Comment (polybuildr) [inline 13/main/index.html]: Alright, we'll track this too if you want. Let's put it in the common notice code.

> You can install multiple instances of the Google Analytics tracking code on your web pages to send data to multiple properties in your account.

from https://support.google.com/analytics/answer/1032400?hl=en

File: .idea/workspace.xml (hunk 1 above)
Comment (aayn) [inline .idea/workspace.xml]: Yes :). I'll remove those.

**Comment by aayn** [comment]
Thanks :). I'll fix the mentioned issues by tomorrow morning.

**Comment by aayn** [comment]
Done. Anything else?(I haven't done the GA tracking as I don't know what Google account to use for that)

**Comment by polybuildr** [comment]
I haven't reviewed it in any detail yet, but could you please [squash these commits](http://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git/)? It would be nice to not have those things in the git history at all.

**Comment by polybuildr** [comment]
Assuming the close isn't a mistake: you can keep this PR open, update the relevant commit, the PR will get updated.

**Comment by aayn** [comment]
Uh oh. I was trying the squashing(my first time) and this PR got closed.
Now, I do have just one commit on my forked repo. What should I do next?

**Comment by aayn** [comment]
I think I found my mistake. Gimme 2 minutes.

**Comment by polybuildr** [comment]
Sure. :) Squashing is a little weird the first few times. Also look at `git reset --soft` and `git commit --amend`, both very useful. (The second answer on that StackOverflow link gives details.)

Do tell us if you have trouble figuring it out.

**Comment by aayn** [comment]
The 'git reset --soft HEAD~3' throws an 'ambiguity' error. So, I'm trying something else.

**Comment by polybuildr** [comment]
I'm not sure here, but that might be because there is no `HEAD~3` (3 commits before HEAD) here, since these are the first few commits (2, I think?).

But it any case, since there were no commits before this, it's generally a weird case, as there's technically no commit to revert too. Sometimes the easy way out is to get rid of `.git` and just start again. (I did not say that. :P)
